docs/configuration.md

```diff
@@ -46,6 +46,10 @@ BMO_CONCURRENCY value lower than the requested PROVISIONING_LIMIT. Default is 20
 image for nodes that use IPv6. In dual stack environments, this can be
 used to tell Ironic which IP version it should set on the BMC.
 
+`PROVISIONING_NETWORK_DISABLED` -- Set to `true` if your deployment does not
```

Comment:
I would suggest expanding on `PROVISIONING_NETWORK` term, Redfish can also use `provisioning network` but idea here is to facilitate deprecation of IPMI is I understand it

Reply:
ok, so maybe list should be expanded to have also other drivers as example, to make the term a bit more clearer.

Reply:
yea, lets just change that to something like `drivers like IPMI, redfish but not redfish-virtualmedia`